pkg/systemd/services.jsx

```diff
@@ -921,15 +922,18 @@ const ServicesPage = () => {
                         {activeTab == "timer" && owner == "system" && superuser.allowed && <CreateTimerDialog isLoading={isLoading} owner={owner} />}
                     </Flex>
                 </PageSection>}
-                <ServicesPageBody
+                { activeTab == "boot"
+                    ? <BootInfo />
+                    : <ServicesPageBody
```

Comment:
This feels bolted on -- all other tabs are unit types, this one is completely different. I could live with it, though.

Perhaps we should instead have a separate "Boot chart" button? @garrett any opinion/idea?

Reply:
I understand what you mean, but I wouldn't know where else to put it. @garrett do you have a suggestion?